### PR TITLE
Document options for building static assets.

### PR DESCRIPTION
**Pull request by jmcarp**
cc @afeld @dlapiduz

### CODE REVIEW

File: content/apps/multi-buildpack-deploys.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ title: Multi-Language Projects
 
 CloudFoundry is capable of deploying, multi-language projects by using a special [buildpack](https://github.com/ddollar/heroku-buildpack-multi). In short, this buildpack applies any buildpacks listed in `.buildpacks` file.
 
-## Preparing and app for a multi-buildpack deploy
+Note: If you're investigating multi-buildpack deploys to build static assets on cloud.gov, you can avoid this issue by [building assets on CI]({{< relref "assets.md#build-ci" >}}#).
```
Comment (afeld) [inline content/apps/multi-buildpack-deploys.md]: Not a blocker, but we might want to add some more to the top of this page to discourage people from using multi-buildpacks, e.g.

> If you are using a multi-buildpack to run multiple long-running processes, you should run them as separate Cloud Foundry applications.

**Comment by afeld** [comment]
Nice! Thanks for writing this up.

File: content/apps/multi-buildpack-deploys.md (hunk 1 above)
Comment (jmcarp) [inline content/apps/multi-buildpack-deploys.md]: Thanks, added that text.

**Comment by dlapiduz** [comment]
Beautiful @jmcarp thanks for writing it up!

**Pull request merged**